### PR TITLE
Eliminating axios dependency and adds further example

### PR DESCRIPTION
**Pull request by IllogicalTree**
Hey guys awesome project, really liked the use of the builder pattern! Doubt if yous were expecting any external pull requests but.. surprise!

In short, I thought it might be better to reduce the number of external dependencies in favour of the built in fetch api, this can be seen in the npm package and express server example. I found it a bit strange that yous weren't making use of the wrapper library in your own express server example, hence that addition. All functionality should remain identical.

Appreciate you perhaps weren't looking for contributions, so no hard feelings if this is rejected :)

### CODE REVIEW

**Comment by AdamGovier** [comment]
Thanks Jordan! Good point on reducing dependencies.  Fetch is new to node. I'm aware that it's been in the browser a long while just forgotten that it has been added to node. I will ask @FrancescoCoding to merge. Cheers!

**Comment by AdamGovier** [comment]
Also @FrancescoCoding worked on the express server before I made the module, so that was a great idea to apply the module to the original project. Thanks very much!

**Comment by FrancescoCoding** [comment]
> Hi @IllogicalTree thank you for your PR! May I ask you to create a new branch for this so I can pull it to test all the changes? 😎

Actually nevermind @IllogicalTree I can just pull the PR as a branch, didn't know that! I'll get back to you once I'm done reviewing

File: express_server/server.js [hunk 1]
```diff
@@ -19,12 +19,12 @@ app.post("/askgpt", async (req, res) => {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
-        "Authorization": `Bearer ${OPENAI_API_KEY}`,
+        Authorization: `Bearer ${OPENAI_API_KEY}`,
```
Comment (IllogicalTree) [inline express_server/server.js:22]: Whats the reason behind this change?

File: express_server_easyGpt/server.js [hunk 2]
```diff
@@ -12,26 +12,26 @@ const OPENAI_API_KEY = process.env.OPENAI_API_KEY; // Your OpenAI API Key
...
 
-  messages.forEach(message => {
-    gpt.addMessage(message.content, message.role)
-  })
+  messages.forEach((message) => {
+    gpt.addMessage(message.content, message.role);
+  });
 
-  console.log(gpt)
+  // console.log(gpt);
```
Comment (IllogicalTree) [inline express_server_easyGpt/server.js:24]: Probably makes more sense to remove this entirely as opposed to commenting it out?

File: express_server/server.js (hunk 1 above)
Comment (FrancescoCoding) [reply express_server/server.js:22]: Content-Type has a dash, and hence it needs to be wrapped :^) ""

File: express_server_easyGpt/server.js (hunk 2 above)
Comment (FrancescoCoding) [reply express_server_easyGpt/server.js:24]: Could do!

**Review by FrancescoCoding — APPROVED** [review]
Did a few changes and some formatting. Looks good now. GG @IllogicalTree

**Pull request merged**